tests/prepare/feature/test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    for method in ${PROVISION_METHODS:-"virtual"}; do
+	# EPEL
+        rlPhaseStartTest "Enable EPEL"
+            rlRun -s "tmt run plan --name epel-enable provision --how $method prepare"
+            rlAssertGrep 'Repo epel.*enabled' "$rlRun_LOG"
+
+            rlRun -s "tmt run plan --name epel-disable provision " \
+                     "--how $method prepare --how feature --epel enabled"
+            rlAssertGrep 'Repo epel.*enabled' "$rlRun_LOG"
+        rlPhaseEnd
+
+        rlPhaseStartTest "Disable EPEL"
+            rlRun -s "tmt run plan --name epel-disable provision --how $method prepare"
+            rlAssertGrep 'Repo epel.*disabled' "$rlRun_LOG"
+
+            rlRun -s "tmt run plan --name epel-enable provision " \
+                     "--how $method prepare --how feature --epel disabled"
+            rlAssertGrep 'Repo epel.*disabled' "$rlRun_LOG"
+        rlPhaseEnd
+
+	# FIPS
+        rlPhaseStartTest "Enable FIPS"
+            rlRun -s "tmt run plan --name fips-enable provision --how $method prepare"
+
+            rlRun -s "tmt run plan --name fips-disable provision " \
+                     "--how $method prepare --how feature --fips enabled"
```

Comment:
So the second case above is running the same test (check for enabled) against a disabled fips setup, so I guess this cannot work like this?